src/aleph_client/commands/instance/__init__.py

```diff
@@ -287,33 +299,52 @@ def validate_ssh_pubkey_file(file: Union[str, Path]) -> Path:
         if crn and (confidential or not hold):
             if not crn.url:
                 return
+            # price: PriceResponse = await client.get_program_price(item_hash) // https://github.com/aleph-im/aleph-sdk-python/pull/143
+            price = "0.000003555555555555"  # This value work only because right now PriceCalculations is bug on this release (patch already in pyaleph)
```

Comment:
Need `https://github.com/aleph-im/aleph-sdk-python/pull/143` merge to be able to use PriceResponse instead of hard coding price.